app/src/main/java/com/example/splendormobilegame/activities/GameActivity/DeckReservingController.java

```diff
@@ -23,15 +32,25 @@ protected DeckReservingController(T activity, TurnController turnController) {
         this.reservationFromDeckMessageHandler = new ReservationFromDeckMessageHandler();
     }
 
-    public void reserveCard(int cardTier) {
+    public void reserveCard(UUID cardUuid) {
```

Comment:
This function should receive `int cardTier` as in the issue description was said.